website/content/docs/upgrading/upgrade-specific.mdx

```diff
@@ -44,6 +51,14 @@ This change removes the backward-compatibility behavior introduced in Consul 1.1
 
 ## Consul 1.15.x
 
+#### Vault Enterprise as CA
+Using Vault as CA with Consul version 1.15.6 will fail to initialize the CA if [`namespace`](/consul/docs/connect/ca/vault#namespace) is set
+but [`intermediate_pki_namespace`](/consul/docs/connect/ca/vault#intermediatepkinamespace) or [`root_pki_namespace`](/consul/docs/connect/ca/vault#rootpkinamespace)
+are empty. This is a bug which will be fixed in a future version.
+
```

Comment:
nit: 1.16 and 1.14 sections have all these sentences in 1 paragraph, but 1.15 breaks it into 2 because of this empty line